Index: Pokemon/src/main/java/tp1/clases/errores/Error.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package tp1.clases.errores;\n\npublic abstract class Error {\n    private String mensaje;\n    public Error(String mensaje) {\n        this.mensaje = mensaje ;\n    }\n\n\n    public void mostrar() {\n        System.out.println(mensaje) ;\n<<<<<<< HEAD\n\n=======\n>>>>>>> main\n    }\n\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Pokemon/src/main/java/tp1/clases/errores/Error.java b/Pokemon/src/main/java/tp1/clases/errores/Error.java
--- a/Pokemon/src/main/java/tp1/clases/errores/Error.java	(revision 7b00fd555783969dc7a6eaf6258a4268886fad00)
+++ b/Pokemon/src/main/java/tp1/clases/errores/Error.java	(date 1697396323142)
@@ -1,18 +1,13 @@
 package tp1.clases.errores;
 
 public abstract class Error {
-    private String mensaje;
+    private final String mensaje;
     public Error(String mensaje) {
-        this.mensaje = mensaje ;
+        this.mensaje = mensaje;
     }
 
-
     public void mostrar() {
-        System.out.println(mensaje) ;
-<<<<<<< HEAD
-
-=======
->>>>>>> main
+        System.out.println(mensaje);
     }
 
 }
